packages/babel-plugin-ember-test-metadata/__tests__/utils-test.js

```diff
@@ -64,19 +69,71 @@ describe('Unit | utils | getNormalizedFilePath', () => {
       filename: path.join('this', 'is', 'not-an-embroider', 'path'),
     },
   };
+  const projectInfo = {
+    pkg: {
+      name: 'test-app',
+      'ember-addon': {},
+    },
+  };
 
   it('returns stripped file path as expected', () => {
-    expect(getNormalizedFilePath(fileOpts.embroiderBuildPath)).toBe(
+    expect(getNormalizedFilePath(fileOpts.embroiderBuildPath, projectInfo)).toBe(
       path.join('tests', 'acceptance', 'my-test.js')
     );
-    expect(getNormalizedFilePath(fileOpts.embroiderBuildPathTwoEmbroiderTokens)).toBe(
+    expect(getNormalizedFilePath(fileOpts.embroiderBuildPathTwoEmbroiderTokens, projectInfo)).toBe(
       path.join('tests', 'acceptance', 'my-test.js')
     );
   });
 
   it('returns unmodified file path when path does not include "embroider" as a segment', () => {
-    expect(getNormalizedFilePath(fileOpts.normalFilePath)).toBe(
+    expect(getNormalizedFilePath(fileOpts.normalFilePath, projectInfo)).toBe(
       path.join('this', 'is', 'not-an-embroider', 'path')
     );
   });
 });
+
+describe('Unit | utils | getParsedClassicFilepath', () => {
```

Comment:
This is a huge nit, but can we not use these types of strings for our suite names? This is very much an Ember/custom convention, not one we should use elsewhere. I think just doing

```suggestion
describe('getParsedClassicFilepath', () => {
```

would be fine.